docs/GettingStarted.md

```diff
@@ -260,6 +260,20 @@ Follow the [Watchman installation guide](https://facebook.github.io/watchman/doc
 > [Watchman](https://facebook.github.io/watchman/docs/install.html) is a tool by Facebook for watching
 changes in the filesystem. It is highly recommended you install it for better performance, but it's alright to skip this if you find the process to be tedious.
 
+### Inotify watchers (Linux Only)
```

Comment:
You should be able to gate this paragraph to be Linux-only using a `<block />` tag.